### ISSUE TITLE
PSR Indicator (*) When Jumping Over Water.

### ISSUE THREAD

**Issue by BLOODWOLF333** [MegaMek/megamek#1035]
### Environment
MegaMek 0.44.0
Java version 1.8.0_144
Platform Windows 8.1 6.3 (amd64)

### Description
It doesnt cause a PSR but the movement path overlay text displays a * when jumping over water hexes.
![github](https://user-images.githubusercontent.com/16950832/42484722-fa9e740c-83b9-11e8-999f-07271c220aec.png)

### Files
[megameklog.txt](https://github.com/MegaMek/megamek/files/2178568/megameklog.txt)
[JumpPSR.sav.gz](https://github.com/MegaMek/megamek/files/2178569/JumpPSR.sav.gz)

**Comment by SBBurzmali** [comment]
I don't think that is a PSR flag, I think it is a "this move is impossible" flag as moving into the water would be an illegal move. The issue is that it is persistent, so it shows even when the full move is valid, and it only shows when the level the water is on is the same as the previous step's level, jump over lower water won't trigger this.

**Comment by rjhancock** [comment]
Closing due to age. Please test with the latest milestone, 0.50.11, to see if the issue still persists. If it does, please re-open.
